### PR TITLE
Add GitHub actions:CI, PR and release

### PR DESCRIPTION
**Pull request by vparfonov**
Add GitHub actions for:
- CI
-  Pull Request to the master branch 
-  release

### CODE REVIEW

File: .github/workflows/pr.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    branches: [ master ]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: "Checkout workspace-data-sync source code"
+      uses: actions/checkout@v2
+    - name: "Run build script"
+      run:  ./build.sh
+
```
Comment (ibuziuk) [inline .github/workflows/pr.yml:22]: no new line

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -1,33 +1,32 @@
...
     branches: [ master ]
-  pull_request:
-    branches: [ master ]
-
-# A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
-  # This workflow contains a single job called "build"
   build:
-    # The type of runner that the job will run on
     runs-on: ubuntu-latest
```
Comment (ibuziuk) [inline .github/workflows/ci.yml:17]: can we use centos?

Comment (vparfonov) [reply .github/workflows/ci.yml:17]: i will check

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+# SPDX-License-Identifier: EPL-2.0
+#
+name: RELEASE
+# Triggers the workflow on push  to the release branch
+on:
+  push:
+    branches: [ release ]
+jobs:
+  build:
+    runs-on: ubuntu-latest
```
Comment (ibuziuk) [inline .github/workflows/release.yml:16]: can we use centos?

**Review by ibuziuk — APPROVED** [review]
left a couple of comments, in general LGTM

File: .github/workflows/ci.yml (hunk 2 above)
Comment (vparfonov) [reply .github/workflows/ci.yml:17]: looks like `centos` not available https://docs.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners#supported-runners-and-hardware-resources

File: .github/workflows/release.yml (hunk 3 above)
Comment (vparfonov) [reply .github/workflows/release.yml:16]: looks like centos not available https://docs.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners#supported-runners-and-hardware-resources

**Pull request merged**